tsconfig.json

```diff
@@ -22,5 +22,6 @@
       "types": ["types"]
     }
   },
-  "exclude": ["**/*.test.ts", "test", "**/__test__"]
+  "include": ["src/**/*.ts", "src/swagger.json"],
+  "exclude": ["**/*.test.ts", "test", "**/__test__", "**/__tests__"]
```

Comment:
I assume an accidental duplication of `**/__test__`

Reply:
> Hang on why have we got inconsistent folder names

I don't know, we have test and test**s** :)

Is that ok ?

Reply:
Before it was just the string "**/__test__"  .. noticed there there are a couple of folders named test**s** so added __tests__ .. after checking this again it looks like there are no folders with the name __test__ so i guess i can changed it to :

```
"exclude": ["**/*.test.ts", "test", "**/__tests__"],
```

is that ok ?

Reply:
I'd prefer to rename the `__tests__` folders `__test__` if I'm honest

Reply:
yes that's fine